workflows/phaseimpute/main.nf

```diff
@@ -100,32 +109,41 @@ workflow PHASEIMPUTE {
     // Prepare panel
     //
     if (params.step == 'impute' || params.step == 'panel_prep' || params.step == 'validate' || params.step == 'all') {
-        // Remove if necessary "chr"
+        // Check chr prefix and remove if necessary
         VCF_CHR_CHECK(ch_panel, ch_fasta)
         ch_versions = ch_versions.mix(VCF_CHR_CHECK.out.versions)
 
-        // Prepare the panel
-        GET_PANEL(VCF_CHR_CHECK.out.vcf, ch_fasta)
-        ch_versions = ch_versions.mix(GET_PANEL.out.versions)
-        ch_panel_sites_tsv = GET_PANEL.out.panel
-            .map{ metaPC, norm, n_index, sites, s_index, tsv, t_index, phased, p_index
-                -> [metaPC, sites, tsv]
-            }
-        CONCAT_PANEL(GET_PANEL.out.panel
-            .map{ metaPC, norm, n_index, sites, s_index, tsv, t_index, phased, p_index
-                -> [[id:metaPC.panel], sites, s_index]
-            }
+        // Normalize indels in panel
+        VCF_NORMALIZE_BCFTOOLS(VCF_CHR_CHECK.out.vcf, ch_fasta)
```

Comment:
Don't forget to add the versions of the software to the ch_versions channel as well as usefull stats and files to the multiqc channel ? 😉